depthmapXcli/main.cpp

```diff
@@ -46,8 +46,8 @@ int main(int argc, char *argv[])
     }
     catch( std::exception &e)
     {
-        std::cout << e.what() << "\n";
-        args.printHelp();
+        std::cout << e.what() << std::endl
```

Comment:
That first `std::endl` could/should probably still be a `\n` - you only need to flush once, after all.